components/connect-github.tsx

```diff
@@ -2,29 +2,17 @@ import { useTranslation } from "next-i18next";
 
 import GithubImage from "components/github-image";
 
-import { useAuthentication } from "contexts/authentication";
-
-
-import Button from "./button";
+import { useAuthentication } from "x-hooks/use-authentication";
 
 interface IProps{
   size?: 'md' | 'sm';
 }
 
 export default function ConnectGithub({size = 'md'}:IProps) {
   const { t } = useTranslation("common");
-
-  const { wallet, connectGithub, isConnecting } = useAuthentication();
+  const { connectGithub } = useAuthentication();
 
 
-  if(size === 'sm'){
-    return (
-    <Button onClick={connectGithub} disabled={isConnecting} isLoading={isConnecting}> 
-      <GithubImage  opacity={1} /> 
-      <span>{t("actions.connect")}</span>
-    </Button>)
-  }
-
```

Comment:
why this button has been removed?

Reply:
@clarkjoao might have been a miss-rebase will match against master before merging so we don't lose actual work :)